python/ambassador/envoy/v2/v2httpfilter.py

```diff
@@ -262,6 +262,9 @@ def V2HTTPFilter_authv1(auth: IRAuth, v2config: 'V2Config'):
         if 'allow_partial' in raw_body_info:
             body_info['allow_partial_message'] = raw_body_info['allow_partial']
 
+        if raw_body_info.get('body_raw_bytes', False):
+            body_info['pack_as_bytes'] = raw_body_info['body_raw_bytes']
+
```

Comment:
... I'm shocked Envoy supports this in v2.

Reply:
Actually, it's not in `envoy/config/filter/http/ext_authz/v2/ext_authz.proto`. How the heck is this working?

Reply:
Where I'm at is that without an explanation of why it works in v2, I'm thinking that means that the tests aren't doing a good-enough job of actually testing it.